packages/dataclass-editor/src/components/dataclass/detail/dataclass/DataClassClassType.tsx

```diff
@@ -0,0 +1,28 @@
+import { BasicSelect, Collapsible, CollapsibleContent, CollapsibleTrigger } from '@axonivy/ui-components';
+import { useAppContext } from '../../../../context/AppContext';
+import { useDataClassChangeHandlers } from '../../data/dataclass-change-handlers';
+import { classTypeOf } from '../../data/dataclass-utils';
+
+export const DataClassClassType = () => {
+  const { dataClass } = useAppContext();
+  const { handleClassTypeChange } = useDataClassChangeHandlers();
+
+  const classType = classTypeOf(dataClass);
```

Comment:
My thinking was that I could easily unit test these functions instead of having to write playwright tests interacting with the component to see if changes to the data class data get applied correctly behind the scenes.
But I guess I can also write unit tests for it this way, just need to export those functions :+1:

Reply:
you have no also exported the functions :)
And you can definitely write unit tests for this 👍 just use the testing-library stuff